### ISSUE TITLE
Do deformable convolution of feature_x and feature_z.

### ISSUE THREAD

**Issue by noUmbrella** [chengdazhi/Deformable-Convolution-V2-PyTorch#1]
Can this tools do deformable convolution of feature_x and feature_z, that is both them are not learnable parameters, but two tensor. In other words, dose this tool support F.conv2d(x,z) like in pytorch?

**Comment by chengdazhi** [comment]
I don't see how pytorch conv2d can accept two feature tensors as input, according to the class definition: https://github.com/pytorch/pytorch/blob/b740b92f3600840e09d4c93f3138333838d1e474/torch/nn/modules/conv.py#L191 

Do you mean z is the weight?

**Comment by noUmbrella** [comment]
Please refer "torch.nn.functional.conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1, groups=1) " this function.@chengdazhi

**Comment by chengdazhi** [comment]
So the feature_z is actually the conv's weight? Our operator supports this, you can look into the Functions and the Modules. The Function is a shallow wrapper of the cuda kernels, and the weights are defined and initialized in the Modules. You can easily customize the module to fit your needs.
